CHANGELOG.md

```diff
@@ -75,7 +81,7 @@
   `headingLevel: <number>` parameter. Default is `2`.
   ([PR #853](https://github.com/alphagov/govuk-frontend/pull/853))
 
-- Update date input component  
+- Update date input component
```

Comment:
can we keep this in here?

Reply:
Yeah, we need to figure out how to get our editors stripping whitespace consistently, but that should not block this PR